testing/trino-server-dev/etc/catalog/elasticsearch.properties

```diff
@@ -0,0 +1,3 @@
+connector.name=elasticsearch
```

Comment:
I'd put this config change in separate commit

Reply:
Okay. Done.